client/src/components/PublicPages/AuditPlanner/AuditPlanner.test.tsx

```diff
@@ -251,16 +251,16 @@ test('Entering election results - validation and submit', async () => {
     await areExpectedErrorMessagesDisplayed({
       displayed: ['Required'],
     })
-    userEvent.type(numberOfWinnersInput, '3')
+    userEvent.type(numberOfWinnersInput, '2')
     await areExpectedErrorMessagesDisplayed({
-      displayed: ['Cannot be greater than number of candidates'],
+      displayed: ['Must be less than number of candidates'],
```

Comment:
Do we have plans to support vote-for-n contests? This will have to change for that, fwiw.

Reply:
Good q! So we can currently support vote-for-n contests, e.g. 3 candidates, 2 winners (in the UI, there's an option to add candidates - you aren't restricted to 2).

I only changed the logic to prevent the same number of winners as candidates, e.g. 3 candidates, 3 winners, to avoid hitting this error in audit math:

https://github.com/votingworks/arlo/blob/e4b637ee5f96ed2cce7d266ec4f4792ee9434e65/server/audit_math/bravo.py#L356

Reply:
ah this is my bad for forgetting how things work! I don't mean n-winners, I mean vote-for-n (i.e. there may only be one winner, but you can vote for 3 candidates or whatever). I _think_ it should just work as long as we aren't checking that (number of total votes cast) <= (number of ballots) (basically you have to do a multiplicity thing where you actually check (number of votes cast) <= n*(number of ballots), where n is the total number of votes.

Reply:
Ah I see, thanks for clarifying! This is really interesting. We actually do check that the total number of votes is less than the total ballots cast:

<img width="655" alt="error" src="https://user-images.githubusercontent.com/12616928/192848189-197706e3-a0a7-42bf-91fc-d7b50084e33d.png">

To perform the proper multiplied check, we'd need to add an input to the audit planner for number of votes allowed. I know @jonahkagan's original mocks didn't include an input for that, likely to keep things simple. Any thoughts on revisiting the inclusion/exclusion of that input, Jonah?

Reply:
Related to this, because we don't expose an input for number of votes allowed, I'm just setting number of votes allowed to number of winners here:

https://github.com/votingworks/arlo/blob/c6774fd7ca8a3f6fad2cae56e5749048b87e6991/server/api/public.py#L102

I've done a quick spot check to verify that sample size calculations don't actually need number of votes allowed.

Reply:
Opened a discussion thread in Slack since there are a few moving pieces here: https://votingworks.slack.com/archives/CKCVA0F9S/p1664387582424249

Reply:
Short-term: Decided to remove the remove requirement that `total ballots cast >= total number of votes` https://github.com/votingworks/arlo/pull/1600/commits/42a9b20d7257a6d451f0cefc6255316497a30222

Long-term: We'll likely want to add an input for the number of votes allowed. Took note of this in https://github.com/votingworks/arlo/issues/1440#issuecomment-1261352006. Will look into this after this PR!